widget_driver/test/test_helpers/test_container_drivable_widget.dart

```diff
@@ -15,6 +16,12 @@ class TestContainerDriver extends WidgetDriver {
     didCallTestMethod = true;
     notifyWidget();
   }
+
+  int updateCount = 0;
```

Comment:
Maybe name this something more descriptive.. it counts the number of times the `updateDriverProvidedProperties` was called.. Can we work this into into the name?